docs/sources/write/links/index.md

```diff
@@ -0,0 +1,103 @@
+For more information and examples, refer to [`docs/reference` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsreference).
+
+## Destination page is on grafana.com
+
+Use a fully qualified URL with version substitution syntax (if needed).
+Version substitution is necessary for fully qualified URLs to link to the correct version of documentation.
+Usually, this is the current version of documentation.
+
+### Examples
+
+**Link to Grafana documentation**:
+
+Start with `https://grafana.com/docs/grafana/<GRAFANA_VERSION>/`, and add the rest of the URL path.
+For example, to link to the [Developers](https://grafana.com/docs/grafana/latest/developers) page with version substitution,
+use `https://grafana.com/docs/grafana/<GRAFANA_VERSION>/developers/`.
+
+- If you are from other documentation, `<GRAFANA_VERSION>` is substituted with the value of `GRAFANA_VERSION` set in the page's front matter.
+
+  Ensure that you set the appropriate version in the root `_index.md` file for your documentation.
+  The following YAML, merged with the existing front matter in the root `_index.md` file sets `GRAFANA_VERSION` to be `latest` for that page and all child pages.
+
+  ```yaml
+  cascade:
+    GRAFANA_VERSION: latest
+  ```
+
+**Link to Grafana Cloud documentation**:
+
+Grafana Cloud documentation is not versioned so no version substitution syntax is needed.
+Use the fully qualified URL.
+For example, to link to the [Author and run tests](https://grafana.com/docs/grafana-cloud/k6/author-run/) page, use `https://grafana.com/docs/grafana-cloud/k6/author-run/`.
+
+**Link to Mimir documentation**:
+
+Start with `https://grafana.com/docs/grafana/<MIMIR_VERSION>/`, and add the rest of the URL path.
+For example, to link to the [Release notes](https://grafana.com/docs/mimir/latest/release-notes/) page with version substitution,
+use `https://grafana.com/docs/mimir/<MIMIR_VERSION>/release-notes/`.
+
+- If you are linking from Mimir documentation, `<MIMIR_VERSION>` is substituted with the version inferred from the page's URL.
+
+- If you are from other documentation, `<MIMIR_VERSION>` is substituted with the value of `MIMIR_VERSION` set in the page's front matter.
+
+  Ensure that you set the appropriate version in the root `_index.md` file for your documentation.
+  The following YAML, merged with the existing front matter in the root `_index.md` file sets `MIMIR_VERSION` to be `latest` for that page and all child pages.
+
+  ```yaml
+  cascade:
+    MIMIR_VERSION: latest
+  ```
+
+## Destination page is external
+
+Use the fully qualified URL.
+For example, `https://github.com`.
+
+## Anchors
+
+In a reference, you can optionally include an anchor to a heading in the referenced page.
+Specify and anchor at the end of the reference `#` followed by the normalized heading.
+
+Hugo normalizes headings to make anchors.
+To convert a heading to an anchor, Hugo makes the following changes:
```

Comment:
This part is a bit hard for me to parse even knowing exactly what the process should look like. The use of the word "normalized" and describing a hashtag as a "reference" sort of requires me to parse two bits of jargon that the reader might not know and we're not really going to explain before we use them, so I'd say let's remove them if possible. I think what probably makes more sense is to show examples of:

- heading
- heading converted to anchor
- anchor alone as a reference
- anchor at the end of a fully qualified URL

I don't think it hurts to still spell out the steps though. All the examples could come after the steps. But is this what **Hugo's** doing or what **you have to do** to create an anchor?
```suggestion

To convert a heading to an anchor, make the following changes:
```